index.html

```diff
@@ -2326,6 +2327,102 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains a remark expressing a reaction to other content within the page.</p>
+				<p>The relationship of comments to other content SHOULD be exposed as follows:</p>
+				<ol>
+					<li>If the comment refers to other content within the page, the source content SHOULD use <pref>aria-details</pref> to point to the related comment.</li>
+					<li>If the comment is a reply to another comment:
+						<ul>
+							<li>If all ancestor comments are available in the DOM, the reply comment SHOULD be a semantic descendant of the comment it relates to, either as a DOM descendant element, or via <pref>aria-owns</pref>. In this case, user agents MUST automatically compute the level and expose it to assistive technologies. This is the preferred method for defining
+								the comment hierarchy, as it creates a single root comment that can be referred to by commented content via <pref>aria-details</pref>.</li>
+							<li>Alternatively, if all ancestor comments are not in the DOM, such as when comments are paginated, the hierarchical
+								level MAY be indicated via <pref>aria-level</pref>.</li>
+						</ul>
```

Comment:
Suggested closing the `<ul>` on prior line so closing the `<li>` here.

```suggestion
					</li>
```